src/main/resources/jsonSchema/clientSettings.schema.json

```diff
@@ -11,6 +11,30 @@
       "description": "Whether or not the language grammar is case-sensitive.",
       "default": false
     },
+    "lineCommentPrefix": {
```

Comment:
Client config JSON schema updates for language grammar-level comment prefix/suffixes, statement terminator characters, and editor behavior features.